### PR TITLE
Add resource menu - v2

### PR DESCRIPTION
**Pull request by dr15**
- Updates `AddResourceMenu` to use the new ui library

### CODE REVIEW

File: src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx [hunk 1]
```diff
@@ -94,13 +100,15 @@ export default function AddResourceMenu({
...
-      <WMDropdown
-        options={options.map((option) => option)}
-        onSelectedChange={onActionSelect}
-        disabled={loading}
-      >
-        <AddResourceButton loading={loading} />
-      </WMDropdown>
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+        <WMMenu
+          menuButtonClassName={cc([{ [classes.menuButtonOpen]: isMenuOpen }])}
```
Comment (dvira) [inline src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx:105]: FYI 'classcat' can take a single object as well.

File: src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx [hunk 2]
```diff
@@ -1,51 +1,32 @@
...
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { ResourceType, TypeName, ContentItem } from '@walkme/types';
+import cc from 'classcat';
 
+import { WMIconArticle, WMIconPlus, WMIconVideo, WMMenu, WMMenuItemShape } from '@walkme/wm-ui';
```
Comment (dvira) [inline src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx:13]: Should be one line higher :)

**Review by dvira — APPROVED** [review]
NOICE!
Left a few suggestions. good job!

File: src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx (hunk 1 above)
Comment (dr15) [reply src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx:105]: Done

File: src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx [hunk 3]
```diff
@@ -94,13 +100,15 @@ export default function AddResourceMenu({
...
   return (
     <>
-      <WMDropdown
-        options={options.map((option) => option)}
-        onSelectedChange={onActionSelect}
-        disabled={loading}
-      >
-        <AddResourceButton loading={loading} />
-      </WMDropdown>
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
```
[the remark this answers is not in the document]
Comment (dr15) [reply src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx:103]: Done

File: src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx (hunk 2 above)
Comment (dr15) [reply src/components/Screen-v2/CourseEditorScreen/ResourceList/AddResourceMenu/index.tsx:13]: Done

**Pull request merged**